[2024-04-03T08:51:46.139+0000] {processor.py:161} INFO - Started process (PID=94) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:51:46.141+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T08:51:46.145+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:51:46.145+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:51:46.156+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:51:46.154+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T08:51:46.157+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:51:46.197+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.066 seconds
[2024-04-03T08:52:16.876+0000] {processor.py:161} INFO - Started process (PID=150) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:52:16.878+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T08:52:16.882+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:52:16.881+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:52:16.890+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:52:16.888+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T08:52:16.890+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:52:16.932+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.064 seconds
[2024-04-03T08:52:47.527+0000] {processor.py:161} INFO - Started process (PID=207) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:52:47.529+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T08:52:47.534+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:52:47.533+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:52:47.541+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:52:47.539+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T08:52:47.542+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:52:47.584+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.064 seconds
[2024-04-03T08:53:18.203+0000] {processor.py:161} INFO - Started process (PID=265) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:53:18.205+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T08:53:18.209+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:53:18.209+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:53:18.216+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:53:18.214+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T08:53:18.217+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:53:18.254+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.058 seconds
[2024-04-03T08:53:48.516+0000] {processor.py:161} INFO - Started process (PID=323) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:53:48.519+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T08:53:48.523+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:53:48.522+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:53:48.531+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:53:48.529+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T08:53:48.531+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:53:48.580+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.072 seconds
[2024-04-03T08:54:18.876+0000] {processor.py:161} INFO - Started process (PID=381) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:54:18.879+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T08:54:18.883+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:54:18.883+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:54:18.891+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:54:18.889+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T08:54:18.892+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:54:18.933+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.065 seconds
[2024-04-03T08:54:49.229+0000] {processor.py:161} INFO - Started process (PID=439) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:54:49.232+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T08:54:49.236+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:54:49.236+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:54:49.244+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:54:49.242+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T08:54:49.245+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:54:49.285+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.065 seconds
[2024-04-03T08:55:19.517+0000] {processor.py:161} INFO - Started process (PID=497) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:55:19.520+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T08:55:19.524+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:55:19.523+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:55:19.531+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:55:19.529+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T08:55:19.532+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:55:19.573+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.064 seconds
[2024-04-03T08:55:49.847+0000] {processor.py:161} INFO - Started process (PID=555) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:55:49.850+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T08:55:49.854+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:55:49.853+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:55:49.860+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:55:49.859+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T08:55:49.861+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:55:49.899+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.060 seconds
[2024-04-03T08:56:20.553+0000] {processor.py:161} INFO - Started process (PID=613) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:56:20.556+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T08:56:20.560+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:56:20.560+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:56:20.568+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:56:20.566+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T08:56:20.568+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:56:20.606+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T08:56:52.239+0000] {processor.py:161} INFO - Started process (PID=671) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:56:52.242+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T08:56:52.246+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:56:52.245+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:56:52.254+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:56:52.252+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T08:56:52.255+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:56:52.297+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.070 seconds
[2024-04-03T08:57:22.592+0000] {processor.py:161} INFO - Started process (PID=729) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:57:22.594+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T08:57:22.599+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:57:22.598+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:57:22.607+0000] {logging_mixin.py:188} INFO - [2024-04-03T08:57:22.605+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T08:57:22.607+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T08:57:22.649+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.066 seconds
[2024-04-03T09:01:09.189+0000] {processor.py:161} INFO - Started process (PID=45) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:01:09.194+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:01:09.198+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:01:09.197+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:01:09.205+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:01:09.203+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:01:09.206+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:01:09.261+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.085 seconds
[2024-04-03T09:01:39.485+0000] {processor.py:161} INFO - Started process (PID=54) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:01:39.488+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:01:39.492+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:01:39.491+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:01:39.499+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:01:39.497+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:01:39.500+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:01:39.537+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.060 seconds
[2024-04-03T09:02:09.793+0000] {processor.py:161} INFO - Started process (PID=63) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:02:09.795+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:02:09.798+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:02:09.798+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:02:09.815+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:02:09.814+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:02:09.816+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:02:09.865+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.079 seconds
[2024-04-03T09:03:28.329+0000] {processor.py:161} INFO - Started process (PID=38) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:03:28.332+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:03:28.336+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:03:28.335+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:03:28.342+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:03:28.340+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:03:28.342+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:03:28.395+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.073 seconds
[2024-04-03T09:03:58.644+0000] {processor.py:161} INFO - Started process (PID=47) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:03:58.646+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:03:58.649+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:03:58.649+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:03:58.657+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:03:58.655+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:03:58.657+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:03:58.697+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:04:28.933+0000] {processor.py:161} INFO - Started process (PID=56) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:04:28.935+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:04:28.939+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:04:28.938+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:04:28.946+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:04:28.944+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:04:28.947+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:04:28.986+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T09:04:59.222+0000] {processor.py:161} INFO - Started process (PID=65) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:04:59.224+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:04:59.228+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:04:59.227+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:04:59.235+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:04:59.233+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:04:59.236+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:04:59.274+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.060 seconds
[2024-04-03T09:05:29.520+0000] {processor.py:161} INFO - Started process (PID=74) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:05:29.522+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:05:29.526+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:05:29.525+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:05:29.533+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:05:29.531+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:05:29.534+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:05:29.574+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:05:59.816+0000] {processor.py:161} INFO - Started process (PID=83) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:05:59.818+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:05:59.822+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:05:59.821+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:05:59.829+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:05:59.827+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:05:59.830+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:05:59.870+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:06:30.114+0000] {processor.py:161} INFO - Started process (PID=92) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:06:30.117+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:06:30.121+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:06:30.120+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:06:30.128+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:06:30.126+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:06:30.129+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:06:30.169+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.063 seconds
[2024-04-03T09:07:00.410+0000] {processor.py:161} INFO - Started process (PID=101) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:07:00.412+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:07:00.416+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:07:00.415+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:07:00.423+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:07:00.421+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:07:00.424+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:07:00.464+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:09:30.350+0000] {processor.py:161} INFO - Started process (PID=39) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:09:30.353+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:09:30.356+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:09:30.356+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:09:30.363+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:09:30.361+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:09:30.364+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:09:30.399+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.056 seconds
[2024-04-03T09:10:00.616+0000] {processor.py:161} INFO - Started process (PID=48) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:10:00.618+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:10:00.622+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:10:00.622+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:10:00.630+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:10:00.628+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:10:00.631+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:10:00.671+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:11:34.857+0000] {processor.py:161} INFO - Started process (PID=45) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:11:34.859+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:11:34.863+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:11:34.863+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:11:34.870+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:11:34.868+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:11:34.870+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:11:34.912+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T09:12:05.130+0000] {processor.py:161} INFO - Started process (PID=54) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:12:05.132+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:12:05.136+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:12:05.135+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:12:05.143+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:12:05.141+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:12:05.144+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:12:05.184+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T09:12:35.428+0000] {processor.py:161} INFO - Started process (PID=63) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:12:35.430+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:12:35.433+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:12:35.433+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:12:35.441+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:12:35.439+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:12:35.441+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:12:35.481+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T09:13:05.719+0000] {processor.py:161} INFO - Started process (PID=72) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:13:05.721+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:13:05.725+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:13:05.724+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:13:05.732+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:13:05.730+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:13:05.733+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:13:05.772+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T09:13:36.013+0000] {processor.py:161} INFO - Started process (PID=81) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:13:36.016+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:13:36.020+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:13:36.019+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:13:36.027+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:13:36.025+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:13:36.028+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:13:36.069+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.064 seconds
[2024-04-03T09:14:06.311+0000] {processor.py:161} INFO - Started process (PID=89) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:14:06.312+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:14:06.316+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:14:06.316+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:14:06.323+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:14:06.322+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:14:06.324+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:14:06.365+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:14:36.607+0000] {processor.py:161} INFO - Started process (PID=98) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:14:36.609+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:14:36.613+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:14:36.612+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:14:36.620+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:14:36.618+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:14:36.621+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:14:36.661+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T09:15:06.899+0000] {processor.py:161} INFO - Started process (PID=107) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:15:06.901+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:15:06.905+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:15:06.904+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:15:06.912+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:15:06.910+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:15:06.913+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:15:06.951+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.060 seconds
[2024-04-03T09:15:37.189+0000] {processor.py:161} INFO - Started process (PID=116) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:15:37.191+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:15:37.195+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:15:37.194+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:15:37.202+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:15:37.200+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:15:37.203+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:15:37.243+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:16:07.493+0000] {processor.py:161} INFO - Started process (PID=125) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:16:07.495+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:16:07.499+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:16:07.498+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:16:07.506+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:16:07.504+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:16:07.507+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:16:07.548+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.063 seconds
[2024-04-03T09:18:16.114+0000] {processor.py:161} INFO - Started process (PID=45) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:18:16.116+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:18:16.120+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:18:16.119+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:18:16.126+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:18:16.125+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:18:16.127+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:18:16.166+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.060 seconds
[2024-04-03T09:18:46.373+0000] {processor.py:161} INFO - Started process (PID=54) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:18:46.375+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:18:46.379+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:18:46.378+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:18:46.386+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:18:46.384+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:18:46.387+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:18:46.430+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.065 seconds
[2024-04-03T09:19:16.671+0000] {processor.py:161} INFO - Started process (PID=63) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:19:16.673+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:19:16.677+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:19:16.676+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:19:16.684+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:19:16.682+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:19:16.685+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:19:16.726+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.063 seconds
[2024-04-03T09:19:46.965+0000] {processor.py:161} INFO - Started process (PID=72) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:19:46.967+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:19:46.971+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:19:46.971+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:19:46.979+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:19:46.977+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:19:46.980+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:19:47.021+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.064 seconds
[2024-04-03T09:20:17.262+0000] {processor.py:161} INFO - Started process (PID=81) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:20:17.264+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:20:17.268+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:20:17.267+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:20:17.275+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:20:17.273+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:20:17.276+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:20:17.316+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.064 seconds
[2024-04-03T09:20:47.555+0000] {processor.py:161} INFO - Started process (PID=90) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:20:47.557+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:20:47.561+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:20:47.560+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:20:47.569+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:20:47.567+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:20:47.570+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:20:47.610+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.063 seconds
[2024-04-03T09:21:17.856+0000] {processor.py:161} INFO - Started process (PID=99) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:21:17.858+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:21:17.862+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:21:17.861+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:21:17.869+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:21:17.867+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:21:17.870+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:21:17.913+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.064 seconds
[2024-04-03T09:21:48.152+0000] {processor.py:161} INFO - Started process (PID=108) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:21:48.154+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:21:48.159+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:21:48.158+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:21:48.170+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:21:48.168+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:21:48.171+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:21:48.910+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:21:48.910+0000] {process_utils.py:262} INFO - Waiting up to 5 seconds for processes to exit...
[2024-04-03T09:23:16.514+0000] {processor.py:161} INFO - Started process (PID=46) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:23:16.517+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:23:16.521+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:23:16.520+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:23:16.527+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:23:16.525+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:23:16.528+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:23:16.566+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.060 seconds
[2024-04-03T09:23:46.785+0000] {processor.py:161} INFO - Started process (PID=55) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:23:46.786+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:23:46.790+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:23:46.789+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:23:46.797+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:23:46.795+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:23:46.798+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:23:46.837+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.060 seconds
[2024-04-03T09:24:17.078+0000] {processor.py:161} INFO - Started process (PID=64) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:24:17.079+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:24:17.083+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:24:17.083+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:24:17.090+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:24:17.088+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:24:17.091+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:24:17.131+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T09:24:47.367+0000] {processor.py:161} INFO - Started process (PID=73) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:24:47.368+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:24:47.372+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:24:47.371+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:24:47.379+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:24:47.377+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:24:47.380+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:24:47.421+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T09:25:17.669+0000] {processor.py:161} INFO - Started process (PID=82) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:25:17.671+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:25:17.675+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:25:17.675+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:25:17.683+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:25:17.682+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:25:17.684+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:25:17.725+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.071 seconds
[2024-04-03T09:27:46.710+0000] {processor.py:161} INFO - Started process (PID=45) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:27:46.713+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:27:46.716+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:27:46.716+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:27:46.723+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:27:46.721+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:27:46.724+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:27:46.761+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.059 seconds
[2024-04-03T09:28:16.994+0000] {processor.py:161} INFO - Started process (PID=54) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:28:16.995+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:28:16.999+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:28:16.998+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:28:17.006+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:28:17.005+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:28:17.007+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:28:17.047+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T09:28:47.286+0000] {processor.py:161} INFO - Started process (PID=63) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:28:47.288+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:28:47.292+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:28:47.291+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:28:47.299+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:28:47.297+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:28:47.300+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:28:47.339+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T09:29:17.571+0000] {processor.py:161} INFO - Started process (PID=72) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:29:17.572+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:29:17.576+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:29:17.575+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:29:17.583+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:29:17.581+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:29:17.584+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:29:17.623+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.060 seconds
[2024-04-03T09:29:47.863+0000] {processor.py:161} INFO - Started process (PID=81) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:29:47.865+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:29:47.869+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:29:47.868+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:29:47.876+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:29:47.874+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:29:47.877+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:29:47.917+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T09:36:04.956+0000] {processor.py:161} INFO - Started process (PID=45) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:36:04.959+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:36:04.963+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:36:04.962+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:36:04.969+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:36:04.968+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:36:04.970+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:36:05.011+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:36:35.232+0000] {processor.py:161} INFO - Started process (PID=54) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:36:35.234+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:36:35.238+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:36:35.237+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:36:35.246+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:36:35.243+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:36:35.246+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:36:35.286+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.064 seconds
[2024-04-03T09:38:36.234+0000] {processor.py:161} INFO - Started process (PID=45) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:38:36.236+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:38:36.240+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:38:36.240+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:38:36.247+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:38:36.245+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:38:36.248+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:38:36.286+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.059 seconds
[2024-04-03T09:39:06.515+0000] {processor.py:161} INFO - Started process (PID=54) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:39:06.517+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:39:06.520+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:39:06.520+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:39:06.528+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:39:06.526+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:39:06.529+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:39:06.573+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.070 seconds
[2024-04-03T09:39:36.811+0000] {processor.py:161} INFO - Started process (PID=63) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:39:36.813+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:39:36.817+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:39:36.817+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:39:36.825+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:39:36.823+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:39:36.826+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:39:36.868+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.065 seconds
[2024-04-03T09:40:07.100+0000] {processor.py:161} INFO - Started process (PID=72) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:40:07.105+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:40:07.109+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:40:07.108+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:40:07.116+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:40:07.114+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:40:07.117+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:40:07.156+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.063 seconds
[2024-04-03T09:40:37.394+0000] {processor.py:161} INFO - Started process (PID=81) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:40:37.396+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:40:37.400+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:40:37.399+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:40:37.407+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:40:37.405+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:40:37.408+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:40:37.447+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.060 seconds
[2024-04-03T09:41:07.681+0000] {processor.py:161} INFO - Started process (PID=90) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:41:07.684+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:41:07.688+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:41:07.688+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:41:07.696+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:41:07.694+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:41:07.697+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:41:07.735+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:41:37.971+0000] {processor.py:161} INFO - Started process (PID=99) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:41:37.972+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:41:37.976+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:41:37.976+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:41:37.983+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:41:37.981+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:41:37.984+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:41:38.025+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:42:08.260+0000] {processor.py:161} INFO - Started process (PID=108) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:42:08.262+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:42:08.266+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:42:08.265+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:42:08.273+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:42:08.271+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:42:08.274+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:42:08.312+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.060 seconds
[2024-04-03T09:42:38.558+0000] {processor.py:161} INFO - Started process (PID=117) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:42:38.560+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:42:38.564+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:42:38.564+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:42:38.571+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:42:38.570+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:42:38.572+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:42:38.612+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:43:08.850+0000] {processor.py:161} INFO - Started process (PID=126) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:43:08.852+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:43:08.856+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:43:08.855+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:43:08.863+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:43:08.861+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:43:08.864+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:43:08.904+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:43:39.183+0000] {processor.py:161} INFO - Started process (PID=135) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:43:39.184+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:43:39.188+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:43:39.188+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:43:39.196+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:43:39.194+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:43:39.196+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:43:39.236+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T09:44:09.476+0000] {processor.py:161} INFO - Started process (PID=144) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:44:09.478+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:44:09.482+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:44:09.481+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:44:09.489+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:44:09.487+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:44:09.490+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:44:09.531+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.063 seconds
[2024-04-03T09:44:39.773+0000] {processor.py:161} INFO - Started process (PID=153) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:44:39.775+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:44:39.779+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:44:39.778+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:44:39.786+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:44:39.784+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:44:39.787+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:44:39.826+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T09:45:10.065+0000] {processor.py:161} INFO - Started process (PID=162) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:45:10.067+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:45:10.071+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:45:10.070+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:45:10.079+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:45:10.076+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:45:10.080+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:45:10.133+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.076 seconds
[2024-04-03T09:49:38.855+0000] {processor.py:161} INFO - Started process (PID=44) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:49:38.858+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:49:38.862+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:49:38.862+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:49:38.877+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:49:38.876+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:49:38.878+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:49:38.915+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.067 seconds
[2024-04-03T09:50:09.123+0000] {processor.py:161} INFO - Started process (PID=53) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:50:09.124+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:50:09.128+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:50:09.128+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:50:09.136+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:50:09.134+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:50:09.136+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:50:09.177+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:50:39.417+0000] {processor.py:161} INFO - Started process (PID=62) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:50:39.419+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:50:39.423+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:50:39.423+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:50:39.436+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:50:39.433+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:50:39.437+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:50:39.492+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.083 seconds
[2024-04-03T09:51:09.722+0000] {processor.py:161} INFO - Started process (PID=71) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:51:09.724+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:51:09.728+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:51:09.727+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:51:09.735+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:51:09.733+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:51:09.736+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:51:09.775+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T09:51:40.012+0000] {processor.py:161} INFO - Started process (PID=80) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:51:40.014+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:51:40.017+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:51:40.017+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:51:40.025+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:51:40.023+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:51:40.026+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:51:40.064+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.060 seconds
[2024-04-03T09:52:10.303+0000] {processor.py:161} INFO - Started process (PID=89) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:52:10.305+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:52:10.309+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:52:10.308+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:52:10.316+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:52:10.314+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:52:10.317+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:52:10.356+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T09:52:40.595+0000] {processor.py:161} INFO - Started process (PID=98) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:52:40.598+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:52:40.601+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:52:40.601+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:52:40.608+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:52:40.607+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:52:40.609+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:52:40.648+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.060 seconds
[2024-04-03T09:53:10.889+0000] {processor.py:161} INFO - Started process (PID=107) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:53:10.891+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:53:10.895+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:53:10.894+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:53:10.902+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:53:10.900+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:53:10.905+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:53:10.943+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T09:53:41.185+0000] {processor.py:161} INFO - Started process (PID=116) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:53:41.187+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:53:41.191+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:53:41.190+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:53:41.198+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:53:41.196+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:53:41.199+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:53:41.239+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:54:11.479+0000] {processor.py:161} INFO - Started process (PID=125) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:54:11.482+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:54:11.485+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:54:11.485+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:54:11.493+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:54:11.491+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:54:11.493+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:54:11.532+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T09:55:55.265+0000] {processor.py:161} INFO - Started process (PID=45) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:55:55.267+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:55:55.271+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:55:55.271+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:55:55.278+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:55:55.276+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:55:55.278+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:55:55.317+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.061 seconds
[2024-04-03T09:56:25.561+0000] {processor.py:161} INFO - Started process (PID=54) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:56:25.563+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:56:25.567+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:56:25.566+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:56:25.574+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:56:25.572+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:56:25.575+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:56:25.615+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:56:55.859+0000] {processor.py:161} INFO - Started process (PID=63) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:56:55.862+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:56:55.866+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:56:55.865+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:56:55.873+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:56:55.871+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:56:55.874+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:56:55.914+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:57:26.145+0000] {processor.py:161} INFO - Started process (PID=72) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:57:26.147+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:57:26.151+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:57:26.150+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:57:26.158+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:57:26.156+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:57:26.159+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:57:26.199+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:57:56.435+0000] {processor.py:161} INFO - Started process (PID=81) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:57:56.437+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:57:56.441+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:57:56.440+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:57:56.448+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:57:56.446+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:57:56.449+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:57:56.490+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:58:26.850+0000] {processor.py:161} INFO - Started process (PID=90) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:58:26.852+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:58:26.856+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:58:26.855+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:58:26.863+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:58:26.861+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:58:26.864+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:58:26.903+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T09:58:57.306+0000] {processor.py:161} INFO - Started process (PID=100) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:58:57.309+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:58:57.313+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:58:57.312+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:58:57.322+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:58:57.320+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:58:57.323+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:58:57.364+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.065 seconds
[2024-04-03T09:59:27.712+0000] {processor.py:161} INFO - Started process (PID=110) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:59:27.715+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:59:27.718+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:59:27.718+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:59:27.726+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:59:27.724+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:59:27.727+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:59:27.779+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.075 seconds
[2024-04-03T09:59:58.115+0000] {processor.py:161} INFO - Started process (PID=120) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:59:58.117+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T09:59:58.121+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:59:58.120+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:59:58.128+0000] {logging_mixin.py:188} INFO - [2024-04-03T09:59:58.126+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T09:59:58.129+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T09:59:58.168+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T10:00:28.398+0000] {processor.py:161} INFO - Started process (PID=129) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T10:00:28.400+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T10:00:28.404+0000] {logging_mixin.py:188} INFO - [2024-04-03T10:00:28.403+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T10:00:28.411+0000] {logging_mixin.py:188} INFO - [2024-04-03T10:00:28.409+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T10:00:28.412+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T10:00:28.452+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.062 seconds
[2024-04-03T10:00:58.717+0000] {processor.py:161} INFO - Started process (PID=138) to work on /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T10:00:58.720+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/example_dag_advanced.py for tasks to queue
[2024-04-03T10:00:58.724+0000] {logging_mixin.py:188} INFO - [2024-04-03T10:00:58.723+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T10:00:58.731+0000] {logging_mixin.py:188} INFO - [2024-04-03T10:00:58.729+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/example_dag_advanced.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_dag_advanced.py", line 77, in <module>
    def _going_to_the_beach() -> dict[str, str]:
TypeError: 'type' object is not subscriptable
[2024-04-03T10:00:58.732+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/example_dag_advanced.py
[2024-04-03T10:00:58.768+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/example_dag_advanced.py took 0.059 seconds
